.github/actions/tp-integration-tests/dist/tp-integration-tests.js

```diff
@@ -0,0 +1,32 @@
+/*
```

Comment:
Should this file have  been moved from `tp-e2e-tests.js`? `tp-e2e-tests.js` still exists